tap_github/scraping.py

```diff
@@ -0,0 +1,76 @@
+"""Utility functions for scraping https://github.com
+
+Inspired by https://github.com/dogsheep/github-to-sqlite/pull/70
+"""
+import requests
+import time
+
+
+def scrape_dependents(response: requests.Response, verbose=False):
+    from bs4 import BeautifulSoup
+
+    soup = BeautifulSoup(response.content, "html.parser")
+    # Navigate through Package toggle if present
+    options = soup.find_all("a", class_="select-menu-item")
+    links = []
+    if len(options) > 0:
+        for link in options:
+            links.append(link["href"])
+    else:
+        links.append(response.url)
+
+    if verbose:
+        print(links)
```

Comment:
I'm not sure what the goal is here, but should this use the logger instead of print? Print will send its output to stdout which should be reserved for the actual records, so I think this would actually break the tap's function.